scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/RuntimeReducerEstimator.scala

```diff
@@ -43,69 +72,134 @@ trait RuntimeEstimationScheme {
    *  the time that a "typical" reducer took in a "typical" job.
    *  Suggested implementation: mean or median.
    */
-  def estimateJobTime(times: Seq[Long]): Option[Long]
+  def estimateJobTime(times: Seq[Double]): Option[Double]
 }
 
 trait BasicRuntimeReducerEstimator extends HistoryReducerEstimator {
   import RuntimeReducerEstimator._
 
+  private val LOG = LoggerFactory.getLogger(this.getClass)
+
   def runtimeEstimationScheme: RuntimeEstimationScheme
 
   def estimateReducers(info: FlowStrategyInfo, history: Seq[FlowStepHistory]): Option[Int] = {
-    val reduceTimes: Seq[Seq[Long]] = getReduceTimes(history)
+    val reduceTimes: Seq[Seq[Double]] = getReduceTimes(history).map(xs => xs.map(_.toDouble))
```

Comment:
Sure, might as well since it's common to both places where getReduceTimes is used